README.md

```diff
@@ -71,6 +72,19 @@ Give them 10 min to write down what it is they would like to discuss in class to
 
 Give them one more chance, they get five more minutes to decide together which three topics they want to be discussed today. They also have to get out the duplicates from the board and prioritize on importance.
 
+## Give broken applications
+
+by [@KevinTss](https://github.com/KevinTss)
+
+A good approach to make students more able to get programming concept is to make them just in the shit. Like just let them search and find a way to repair something.
```

Comment:
This sentence is not clear to me. "make them just in the shit"

Reply:
I tried to explain that better